### ISSUE TITLE
Principles by Ray Dalio

### ISSUE THREAD

**Issue by quinnliu** [quinnliu/reading_notes#11]
[Video how the economic machine works](https://youtu.be/PHe0bXAIuk0)

**Comment by quinnliu** [comment]
up to Part 2 page 3

**Comment by quinnliu** [comment]
- I believe there are an infinite number of laws of the universe and that all progress or dreams achieved come from operating in a way that's consistent with them. These laws and the principles of how to operate in harmony with them have always existed. We were given these laws by nature.

**Comment by quinnliu** [comment]
- While most others seem to believe that learning what we are taught is the path to success, I believe that figuring out for urself what u want and how to get it is a better path
- while most others seem to believe that having answers is better than having questions, I believe that having questions is better than having answers because it leads to more learning
- while most others seem to believe that mistakes are bad things, I believe mistakes are good things because I believe that most learning comes via making mistakes and reflecting on them
- while most others seem to believe that finding out about one's weaknesses is a bad thing, I believe that it is a good thing because it is the first step toward finding out what to do about them and not letting them stand in ur way.
- while most others seem to believe that pain is bad, I believe that pain is required to become stronger

**Comment by quinnliu** [comment]
- success is achieved by people who deeply understand reality and know how to use it to get what they want. The converse is also true: idealists who are not well-grounded in reality create problems, not progress. For example, communism was a system created by people with good intentions who failed to recognize that their idealistic system was inconsistent with human nature. As a result, they caused more harm than good.

**Comment by quinnliu** [comment]
- This brings me to my most fundamental principle:
   <b>More precisely, an accurate understanding of reality is the essential foundation for producing good outcomes</b>

**Comment by quinnliu** [comment]
- it seems to me that, in relation to nature, man has the intelligence of a mold growing on an apple- man can't even make a misquito, let alone scratch the surface of understanding the universe.

**Comment by quinnliu** [comment]
- <b>I believe that evolution, which is the natural movement toward better adaptation, is the greatest single force in the universe, and that it is good</b>
- <b>I believe that the desire to evolve, i.e., to get better, is probably humanity's most pervasive driving force.
- Because of the law of diminishing returns, it is only natural that seeking something new, or seeking new depths of something old, is required to bring us satisfaction.
- In other words, the sequence of 1) seeking new things (goals) 2) working and learning in the process of pursuing these goals 3) obtaining these goals and 4) then doing this over and over again is the personal evolutionary process that fulfills most of us and moves society forward.

**Comment by quinnliu** [comment]
- Self-interest and society's interests are generally symbiotic: more than anything else, it is pursuit of self-interest that motivates people to push themselves to do the difficult things that benefit them and that contribute to society. In return, society reward those who give it what it wants. That is why how much money people have earned is a rough measure of how much they gave society what it wanted - NOT how much they desired to make money.

**Comment by quinnliu** [comment]
- There are many people who have made a lot of money who never made making a lot of money their primary goal. Instead, they simply engaged in the work that they were doing, produced what society wanted, and got rich doing it. And there are many people who really wanted to make a lot of money but never produced what the society wanted and they didn't make a lot of money.

**Comment by quinnliu** [comment]
- For example, highly creative, goal-oriented people who are good at imagining the big picture often can easily get tripped up on the details of daily life, while highly pragmatic, task-oriented people who are great with the details might not be creative.  
- <b>The most important quality that differentiates successful people from unsuccessful people is our capacity to learn and adapt to these things</b>

**Comment by quinnliu** [comment]
- However, typically defensive, emotional reactions i.e. ego barriers stand in the way of this progress. These reactions take place in the part of the brain called the amygdala. As a result of them, most people don't like reflecting on their weaknesses even though recognizing them is an essential step toward preventing them from causing them problems. Most people especially dislike others exploring their weaknesses because it makes them feel attacked, which produces flight or flight reactions; however, having others help one find one's weaknesses is essential because it's very difficult to identify one's own.

**Comment by quinnliu** [comment]
- I believe that life consists of an enormous number of choices that come at us and that each decision we make has consequences, so the quality of our lives depends on the quality of the decisions we make. 

- reality + dreams + determination = a successful life

**Comment by quinnliu** [comment]
- Also, for most people happiness is much more determined by how things turn out relative to their expectations rather than the absolute level of their conditions. For example, if a billionaire loses $200 million he will probably be unhappy, while if someone who is worth $10 thousand unexpectedly gets another $2 thousand, he will probably be happy. This basic principle suggests that you can follow one of two paths to happiness: 1) have high expectations and strive to exceed them 2) lower ur expectations so that they are at or below ur conditions. Most of us choose the first path, which means that to be happy we have to keep evolving.

**Comment by quinnliu** [comment]
- Another principle to keep in mind is that people need meaningful work and meaningful relationships in order to be fulfilled. I have observed this to be true for virtually everyone, and I know that it's true for me. Regardless of other's principles, u will need to decide for urself what u want and go after it in the best way for u

**Comment by quinnliu** [comment]
- <b> It is a fundamental law of nature that to evolve one has to push one's limits, which is painful, in order to gain strength - whether it's in the form of lifting weights, facing problems head-on, or in any other way. 
- if u can develop a knee-jerk reaction to pain that is to reflect rather than to fight or flee, it will lead to ur rapid learning/evolving.

**Comment by quinnliu** [comment]
- <b>How much do u let what u wish to be true stand in the way of seeing what is really true?</b>

**Comment by quinnliu** [comment]
- <b>This expalins why people who are interested in making the best possible decisions rarely are confident that they have the best possible answers.

**Comment by quinnliu** [comment]
- People who overweigh the 1st order consequences of their decisions and ignore the effects that the second and subsequent order consequences will have on their goals rarely reach their goals. This is because the first-order consequences often have opposite desirabilities from second-order consequences, resulting in big mistakes in decision-making. For example, the first-order consequences of exercise (pain and time-sink) are commonly considered undesirable, while the 2nd order consequences better health and more attractive appearance are desirable. Similarly, food that tastes good is often bad for u and vice versa, etc. If ur goal is to get physically fit and u don't ignore the 1st order consequences of exercise and good-tasting but unhealthy food and connect ur decisions with their second and 3rd order consequences, u will not reach ur goal

**Comment by quinnliu** [comment]
- Quite often the 1st order consequences are the temptations that cost us what we really want, and sometimes they are barriers that stand in our way of getting what we want. It's almost as though the natural selection process sorts us by throwing us trick choices that have both types of consequences and penalizing the dummies who make their decisions just on the basis of the 1st order consequences alone. 
- By contrast, people who choose what they really want, and avoid the temptations and get over the pains that drive them away from what they really want, are much more likely to have successful lives.

**Comment by quinnliu** [comment]
- People who blame bad outcomes on anyone or anything other than themselves are behaving in a way that is at variance with reality, and subversive to their progress. Blaming bad outcomes on anyone or anything other than one's self is essentially wishing that reality is different than it is, which is silly. And it is subversive because it diverts one's attention away from mustering up the personal strength and other qualities that are required to produce the best possible outcomes.

**Comment by quinnliu** [comment]
- <b>successful people understand that bad things come at everyone and that it is their responsibility to make their lives what they want them to be by successfully dealing with whatever challenges they face. Successful people know that nature is testing them, and that it is not sympathetic</b>

**Comment by quinnliu** [comment]
- How much do u let urself off the hook rather than hold urself accoutable for ur success?

**Comment by quinnliu** [comment]
In summary, I believe that u can probably get what u want out of life if u can suspend ur ego and take a no-excuses approach to achieving ur goals with open-mindedness, determination, and courage, especially if u rely on the help of people who are strong in areas that ur weak.

**Comment by quinnliu** [comment]
stopped at page 24

**Comment by quinnliu** [comment]
1. have clear goals
2. identify and don't telerate the problems that stand in the way of achieving ur goals
3. accurately diagnose these problems
4. design plans that explicity lay out tasks that will get u around ur problems and on to ur goals
5. implement these plans by doing these tasks

**Comment by quinnliu** [comment]
1) u must approach these as distinct steps rather than blur them together.When setting goals, just set goals; when diagnosing problems just diagnose problems. Blurring the steps leads to suboptimal outcomes because it creates confusion and short-changes the individual steps. Doing each step thoroughly will provide information that will help u do the other steps well, since the process is iterative.

2) each of these five steps requires different talents and disciplines. If u recognize ur weaknesses and design around them.

3) It is essential to approach this process in a very clear-headed, rational way rather than emotionally.

**Comment by quinnliu** [comment]
By and large, life will give u what u deserve and it doesn't give a damn what u "like". So it is up to u to take full responsibility to connect what u want with what u need to do to get it, and then to do those things - which often are difficult but produce good results - so that u'll then deserve to get what u want.

**Comment by quinnliu** [comment]
That's just the way it is, so u might as well accept it. Once u accept that playing the game will be uncomfortable, and u do it for a while, it will become much easier (like it does when getting fit). When u excel at it, u will find ur ability to get what u want thrilling. U'll see that excuses like "that's not easy" are of no value and that it pays to "push through it" at a pace u can handle. Like getting physically fit, the most important thing is that u keep moving forward at whatever pace u choose, recognizeing the consequences of ur actions. When u think that it's too hard, remember that in the long run, doing the things that will make u successful is a lot easier than being unsuccessful. The first-order consequences of escaping life's challenges may seem plasurable in the moment, but the second and third-order consequences of this approach are ur life and, over time, will be painful. With practice, u will eventually play this game like a ninja, with skill and a calm centeredness in the face of adversity that will let u handle most of ur numerous challenges well.

**Comment by quinnliu** [comment]
So it's important to remember: it doesn't really matter if some things are unavailable to u, because the selection of what IS available is so great. That is why many people who had major losses who lost their ability to walk, to see, etc and who didn't narrow-mindedly obsess about their loss but rather openmindedly accepted and enjoyed what remained, had equally happy lives as those who didn't ever have these losses. 

In other words, u can have an enormous amount: much, much more than what u need to have for a happy life. So don't get discouraged by not being able to have everything u want, and for God's sake don't be paralyzed by the choices.

**Comment by quinnliu** [comment]
It is important not to confuse "goals" and "desires"

Goals are the things that u really want to achieve, while desires are things u want that can prevent u from reaching ur goals.

**Comment by quinnliu** [comment]
stopped page 27

**Comment by quinnliu** [comment]
While it can be tempting to react emotionally to problems and seek sympathy or blame others, this accomplishes nothing. Whatever the reasons you have to get over the impediments to succeed. Remember that the pains ur feeling are "growing pains" that will test your character and reward you if u push through them. Try to look at ur problems as a detached observer would. Remember that identifying problems is like finding gems embedded in puzzles; if u solve the puzzles u will get the gems that will make ur life much better.

**Comment by quinnliu** [comment]
- more than anything else, what differentiates people who live up to their potential from those who don't is a willingness to look at themselves and others objectively

**Comment by quinnliu** [comment]
stopped page 36

**Comment by quinnliu** [comment]
I'm not studying for the GRE because I'm not doing it first thing in the morning

**Comment by quinnliu** [comment]
- I also believe that the most important difference between great organizations and bad ones is in how well they manage their feedback loops

**Comment by quinnliu** [comment]
The best advice I can give you is to ask yourself what do you want, then ask 'what is true' and then ask yourself 'what should be done about it' I believe that if you do this you will move much faster towards what you want to get out of life than if you don't!

**Comment by quinnliu** [comment]
page 43 starts listing a table of 200 principles to be used when a problem occurs

**Comment by quinnliu** [comment]
done
